static/js/controllers/associations.controller.js

```diff
@@ -305,11 +302,20 @@ async function separateStudies(associations, traitData, refGen, defaultSex) {
                 studyTypes.push("O")
             }
             studyIDsToMetaData[studyObj.studyID] = { citation: studyObj.citation, reportedTrait: studyObj.reportedTrait, studyTypes: studyTypes, traits: {}, ethnicity: ethnicities != "" ? ethnicities : []}
-            studyIDsToMetaData[studyObj.studyID]['traits'][studyObj.trait] = traitStudyTypes
+            
+        }
+        if (!(studyObj.trait in studyIDsToMetaData[studyObj.studyID]['traits'])) {
+            studyIDsToMetaData[studyObj.studyID]['traits'][studyObj.trait] = {
+                studyTypes: traitStudyTypes,
+                pValBetaAnnoValType: [pvalBetaAnnoValType],
+                superPopulations: superPopulations
+            }
         }
         else {
-            studyIDsToMetaData[studyObj.studyID]['traits'][studyObj.trait] = traitStudyTypes
-            studyIDsToMetaData[studyObj.studyID]['ethnicity'] = Array.from(new Set([...studyIDsToMetaData[studyObj.studyID]['ethnicity'], ...ethnicities]))
+            // also change so that pValueAnno is an identifyier? and betaAnno??
```

Comment:
identifyier -> identifier
Do you still need this comment? was it resolved?

Reply:
yes